csrc/quantization/compressed_tensors/int8_quant_kernels.cu

```diff
@@ -28,15 +28,34 @@ namespace vllm {
 template <typename scalar_t, typename scale_type>
 __global__ void static_scaled_int8_quant_kernel(
     const scalar_t* __restrict__ input, int8_t* __restrict__ out,
-    scale_type scale, const int hidden_size) {
+    scale_type inverted_scale, const int hidden_size) {
   const int tid = threadIdx.x;
   const int token_idx = blockIdx.x;
+  const float4* vectorized = reinterpret_cast<const float4*>(input);
+  const int traverse_space = hidden_size >> 2;
```

Comment:
just divides this by 4(since we're using `float4` we need to do this as we're handling 4x as much work per iteration). Just avoiding the division assembly here.